### DISCUSSION TITLE
amd vulkan hdr 全屏显示错误

### DISCUSSION THREAD

**Discussion by 404gods** [hooke007/mpv_PlayKit#422] [mpv 通用]
大佬好，我是A卡，安装了MPV_lazy，设置了HDR直通，解码为gpu-next和winvk，我现在非全屏窗口的时候播放HDR视频正常，但是一全屏就会画面变灰白（若我事先打开显示器HDR和windows HDR则全屏播放颜色也正常）

**Comment by hooke007** [comment]
a卡问题比较抽象，大概率是驱动

**Reply by 404gods** [reply]
我发现不用DHR直通也会切换全屏就变灰白，只要是播放HDR视频就有这个问题。我只是在标准安装包里默认的mpv.conf文件里修改了以下几个代码：
 vo = gpu-next
 gpu-context = winvk
 hwdec = no
 profile = high-quality
 dither-depth = 10
 deband = yes
其他都是默认的没动
若是选择vo = gpu，则全屏正常，但是颜色效果不好较淡。还是想用gpu-next~~~

**Comment by LJHR1999** [comment]
这个我也是一样6800xt，色彩偏淡，mpv输出的色彩空间和曲线是正确的，需要手动调节显示器色彩空间到bt2020。明明电视都提示hdr内容了，为啥不自动切换色域，还是709，一直搞不明白，如果想直通的话可以mpcvr和madvr代替，都可以自动切换。

**Reply by hooke007** [reply]
https://github.com/mpv-player/mpv/issues/13534
暂定为驱动问题
